contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -22,10 +20,10 @@
 <body>
 <div class="container-fluid">
     <div class="topnav" id="myTopnav">
-        <a href="{{ url_for('contribute.home') }}">Contributions Catalog</a>
-        <a href="{{ url_for('contribute.create') }}">Contribute</a>
+        <a href="{{ url_for('contribute.home') }}"><i class="fa fa fa-home fa-lg"></i> Contributions Catalog</a>
+        <a href="{{ url_for('contribute.create') }}"><i class="fa fa-plus-square fa-lg"></i> Contribute</a>
         <div style="float: right; margin-right: 25px;">
-            <a >Hello, {{ user }}!</a>
+            <a ><i class="fa fa-fw fa-user fa-lg"></i> Hello, {{ user }}!</a>
```

Comment:
Please use the pointer cursor when hovering over the name/icon to make it clear that it'sclickable. Thanks.

Reply:
updated.